src/configure.rs

```diff
@@ -54,14 +48,57 @@ pub fn update_configuration(name: &str, value: &str) {
             table.insert(keys[0].to_string(), Value::Table(updated_values));
         }
 
-        let config_str =
-            toml::to_string_pretty(&table).expect("Failed to serialize the config to string");
-        File::create(&config_path)
-            .and_then(|mut file| file.write_all(config_str.as_ref()))
-            .expect("Error writing starship config");
+        write_configuration(table);
+    }
+}
+
+pub fn toggle_configuration(name: &str, key: &str) {
+    if let Some(table) = get_configuration().as_table_mut() {
+        if let Some(values) = table.get(name).unwrap().as_table() {
```

Comment:
As you mentioned in the PR description I think it would be better if we handled this a bit more gracefully, maybe we could log an error here if the module section is not set in the config file?

Reply:
https://github.com/starship/starship/commit/f1bed89155eb2651cc9c8a0046eac5e0b08e1af1

Reply:
Good catch! It's done. Now it should log `Given module 'winamp' not found in config file` and `exit(1)` in case of error.